addon/components/image-layer.js

```diff
@@ -1,19 +1,43 @@
+import Ember from 'ember';
 import BaseLayer from 'ember-leaflet/components/base-layer';
 
+const {
+  deprecate,
+  isPresent
+} = Ember;
+
 export default BaseLayer.extend({
 
   leafletRequiredOptions: [
-    'imageUrl', 'bounds'
+    'url', 'bounds'
   ],
 
   leafletOptions: [
     'attribution', 'opacity'
   ],
 
   leafletProperties: [
-    'url', 'opacity'
+    'url', 'opacity', 'bounds'
   ],
 
+  init() {
+    const imageUrl = this.get('imageUrl');
```

Comment:
For consistency purposes, do you mind changing this to `let`?
Sorry for being pedantic, but I believe this kind of consistency makes the project easier to read and gives some extra credibility.
I will add a linter plugin that enforces these things soon (https://github.com/DockYard/eslint-plugin-ember-suave) but for now 👀